nois-testnet-004/add-accounts.sh

```diff
@@ -8,6 +8,7 @@ free_amount=20000
 ufree_amount=$(expr $free_amount \* 1000000)
 
 validator_addresses=(
+    "nois1p9tw323xdjp5q3yzuecfahmgrpufmm89z93wpk"
```

Comment:
Doe bash allow comments here, like this?

```suggestion
    # Deployment key
    "nois1p9tw323xdjp5q3yzuecfahmgrpufmm89z93wpk"
    # Validators
```